File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/director_unroll_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'director_unroll'

class MyFoo < Director_unroll::Foo
  def ping
    "MyFoo::ping()"
  end
end

a = MyFoo.new

b = Director_unroll::Bar.new

b.set(a)
c = b.get()

raise RuntimeError if a != c

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/director_unroll_runme.py
import director_unroll


class MyFoo(director_unroll.Foo):

    def ping(self):
        return "MyFoo::ping()"


a = MyFoo()

b = director_unroll.Bar()

b.set(a)
c = b.get()


if not (a.this == c.this):
    print a, c
    raise RuntimeError
